### PR TITLE
Generate: add hooks timeout and env options

### PR DESCRIPTION
**Pull request by Mashimiao**

### CODE REVIEW

**Comment by liangchenye** [comment]
I think we should join 'timeout/args/env' together.
We might have a list of hooks with a same path.

**Comment by Mashimiao** [comment]
> I think we should join 'timeout/args/env' together.

You mean just implement timeout and env adding with current --prestart,--poststart,--poststop?
Or add more options like --prestart-env?

**Comment by q384566678** [comment]
> You mean just implement timeout and env adding with current --prestart,--poststart,--poststop?

I agree with this method.

File: cmd/oci-runtime-tool/generate.go [hunk 1]
```diff
@@ -332,14 +338,36 @@ func setupSpec(g *generate.Generator, context *cli.Context) error {
...
+			g.AddPostStartHook(path, args)
+		}
+	}
+
+	if context.IsSet("hooks-poststart-env") {
+		postStartEnvs := context.StringSlice("hooks-poststart-env")
+		for _, postStartEnv := range postStartEnvs {
+			path, env, err := parseHookEnv(postStartEnv)
+			if err != nil {
+				return nil
```
Comment (q384566678) [inline cmd/oci-runtime-tool/generate.go]: return err

Comment (Mashimiao) [inline cmd/oci-runtime-tool/generate.go]: updated, thanks

**Comment by q384566678** [comment]
LGTM

**Comment by liangchenye** [comment]
LGTM

**Pull request merged**